charts/accountapp/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "accountapp.fullname" . }}
+  labels: {{ include "accountapp.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - name: frontend
+      protocol: TCP
+      targetPort: frontend
+      port: 5000
+    - name: backend
+      protocol: TCP
+      targetPort: backend
+      port: 8080
```

Comment:
Fronted and backend are provided by the same docker image that listen on port 8080

Reply:
🤕